packages/features/schedules/components/Schedule.tsx

```diff
@@ -102,9 +102,12 @@ const CopyButton = ({
     <Dropdown open={open} onOpenChange={setOpen}>
       <DropdownMenuTrigger asChild>
         <Button
-          className={classNames(open && "ring-brand-500 !bg-gray-100 outline-none ring-2 ring-offset-1")}
+          className={classNames(
+            "text-gray-700",
+            open && "ring-brand-500 !bg-gray-100 outline-none ring-2 ring-offset-1"
+          )}
           type="button"
-          tooltip={t("duplicate")}
+          tooltip={`${t("copy_times_to")}...`}
```

Comment:
does that make sense in all languages, to have the dots in the end like that?

Reply:
@CarinaWolli no, this has to be inside the string